Lesson2/SecondHomework.playground/Contents.swift

```diff
@@ -32,15 +45,20 @@ class StackStatistics: Stack {
         super.push(element: element)
     }
     
-    override func pop() -> Int? {
+    /// Removes and returns the top element of the stack. Returns nil when the invoking stack is empty.
+    /// - Complexity: O(1)
+    /// - Returns: The top element of the stack or nil.
+    override func pop() -> T? {
         if !minimalValues.isEmpty {
             minimalValues.removeLast()
         }
         return super.pop()
-        
     }
     
-    func minimumElement() -> Int? {
+    /// Returns the minimum element in the stack. Returns nil when the invoking stack is empty.
```

Comment:
```suggestion
    /// Returns the minimum element in the stack.
```

Typically you describe such details in `Returns` section.

Reply:
corrected the description to the methods